package.json

```diff
@@ -15,7 +15,7 @@
     "@docusaurus/core": "^2.2.0",
     "@docusaurus/preset-classic": "^2.2.0",
     "@easyops-cn/docusaurus-search-local": "^0.33.5",
-    "@mdx-js/react": "^1.6.21",
+    "@mdx-js/react": "^2.1.5",
```

Comment:
See <https://github.com/facebook/docusaurus/issues/4029>

As docusaurus has its down mdx requirement https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-classic/package.json#L35, maybe just remove it here and rely on docusaurus for updates @frenck?

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/1547